From f2ede4696f7c43b28b0a463912b6bbb81ffe816d Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Wed, 29 Jul 2015 09:40:56 +0300
Subject: [PATCH 0287/1240] fix: mpp: fixed uninitialize parameter before
 relocation in mpp code

- Fixed uninitialize parameter before relocation in MPP code
- This patch move the parameter to DATA section

Change-Id: Ifeb39f6aa61e7e488ee2271ea519b2727cd2e513
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/22297
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
---
 arch/arm/cpu/mvebu-common/mpp-bus.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/cpu/mvebu-common/mpp-bus.c b/arch/arm/cpu/mvebu-common/mpp-bus.c
index b379016..f258c4d 100644
--- a/arch/arm/cpu/mvebu-common/mpp-bus.c
+++ b/arch/arm/cpu/mvebu-common/mpp-bus.c
@@ -28,7 +28,7 @@
 #include <asm/arch-mvebu/soc.h>
 #include <asm/arch-mvebu/pinctl.h>
 
-struct mpp_bus buses[MAX_MPP_BUSES];
+struct mpp_bus __attribute__((section(".data"))) buses[MAX_MPP_BUSES];
 
 #define DEBUG
 int mpp_get_bus_id(const char *name)
-- 
1.9.1

